### PR TITLE
Sign up and Login Page

### LINKED ISSUE #8: Add fetch support for signup endpoint on Sign Up page
Adding a fetch call logic to the signup endpoint in the backend API.

### PR DESCRIPTION
**Pull request by aryans1204**

### CODE REVIEW

File: frontend/src/components/Login.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+:root {
```
Comment (aryans1204) [inline frontend/src/components/Login.css]: call this `Login.module.css`, the module is important

File: frontend/src/main.jsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+import App from "./App";
+import "./index.css";
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-)
+    <App></App>
```
Comment (aryans1204) [inline frontend/src/main.jsx]: This change really doesn't matter under React. It's perfectly fine to use `<Component />`

**Comment by aryans1204** [comment]
Looks good to be merged, but please also refactor the code so that the components are created in an Object Oriented Manner as opposed to the functional manner, namely via `class Component_name extends component`

File: frontend/src/components/SignUp.jsx [hunk 3]
```diff
@@ -1,43 +1,44 @@
...
+      },
+      body: JSON.stringify({
+        name: this.state.name,
+        email: this.state.email,
+        password: this.state.password,
+      }),
+    })
       .then((response) => {
         if (response.status === 201) {
           console.log("Sign up successful!");
```
Comment (aryans1204) [inline frontend/src/components/SignUp.jsx:44]: At this point, could you also store the token with `token = response.body.user.tokens[0].token`

File: frontend/src/components/SignUp.jsx [hunk 4]
```diff
@@ -50,59 +51,60 @@ function SignUp() {
...
+              </form>
+            </div>
           </div>
         </div>
       </div>
-    </div>
-  );
+    );
+  }
 }
```
Comment (aryans1204) [inline frontend/src/components/SignUp.jsx:110]: at this point, add in a `export default token`

Comment (jason-701) [reply frontend/src/components/SignUp.jsx:110]: i stored it through sessionStorage.setItem(), does it accomplish the same thing?

Comment (aryans1204) [reply frontend/src/components/SignUp.jsx:110]: Yup, it's the same thing

**Comment by aryans1204** [comment]
LGTM 🚀

**Pull request merged**